tools/who_what_benchmark/README.md

```diff
@@ -11,8 +16,46 @@
 * Validation of text-to-image pipelines. Computes similarity score between generated images:
     * Supports Diffusers library and Optimum-Intel via `Text2ImageEvaluator` class.
 
-The main idea is to compare similarity of text generation between baseline and optimized LLMs.
+### Installation
+Install WWB and its requirements from the source using `pip` or any other package manager. For example,
+
+* `python -m venv eval_env`
+* `source eval_env/bin/activate`
+* `pip install -r requirements.txt`
+* `pip install .`
+
+## Usage
+### Compare Text-generation Models (LLMs)
+```sh
+# Collect ground truth from the baseline Hugging Face Transformer model 
+wwb --base-model microsoft/Phi-3-mini-4k-instruct --gt-data gt.csv --model-type text --hf
+
+# Convert model to Optimum-Intel (quantized to 8-bit by default)
+optimum-cli export openvino -m microsoft/Phi-3-mini-4k-instruct phi-3-openvino
+
+# Measure similarity metric for Optimum-OpenVINO inference backend
+wwb --target-model phi-3-openvino --gt-data gt.csv --model-type text
+
+# Measure similarity metric for OpenVINO GenAI inference backend
+wwb --target-model phi-3-openvino --gt-data gt.csv --model-type text --genai
+```
 
+### Compare Text-to-image models (Diffusers)
+```sh
+# Collect ground truth from the baseline Hugging Face Transformer model 
+wwb --base-model microsoft/Phi-3-mini-4k-instruct --gt-data gt.csv --model-type text --hf
```

Comment:
looks like text-to-image examples missed and accidentally copied llm example for phi3 twice....

Reply:
Sorry, didn't submit the latest version.

Reply:
Fixed